### ISSUE TITLE
Stuck on Building EVDI

### ISSUE THREAD

**Issue by codercodingthecode** [AdnanHodzic/displaylink-debian#136]
It just hangs in there for as long as you can let it be.
Ubuntu Budgie 18.04 Kernel 4.15.0.24.
I tried  the master and  modesetting with no avail.

![image](https://user-images.githubusercontent.com/20182536/42179109-c778d950-7e00-11e8-94ec-901707286f98.png)

**Comment by codercodingthecode** [comment]
Alright, so I solved it.  
I did take a few approaches, so kinda hard to pinpoint which one.

I had secured boot turned on. That was the only way I could get the boot locked into the uefi/biios

I turned off secure boot and ran the original installer. It worked straight out after that.

I also did this, so, after rebooting Im not sure if it is needed, but I will leave here if someone still get stuck. The selected answer might've helped.

https://askubuntu.com/questions/762254/why-do-i-get-required-key-not-available-when-install-3rd-party-kernel-modules
